packages/eslint-config-custom/index.js

```diff
@@ -1,24 +1,27 @@
+const { resolve } = require("node:path");
+const project = resolve(process.cwd(), 'tsconfig.json');
+
 module.exports = {
   parser: "@typescript-eslint/parser",
   extends: [
-    "airbnb",
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:import/errors",
     "plugin:import/warnings",
     "plugin:import/typescript",
-    "plugin:prettier/recommended",
     "prettier",
   ],
   ignorePatterns: [
     "index.js",
     ".eslintrc",
     ".eslintrc.cjs",
-    "vite.*.ts",
```

Comment:
Did you mean to remove this?

Reply:
Yeah, i'm not sure why I ignored linting our config files in the first place. Maybe because we were not linting webpack? :thinking: 

I can revert this change if we do actually want to ignore config files in linting though.